bin/posydon-setup-grid

```diff
@@ -78,8 +79,8 @@ def find_inlist_from_scenario(source, gitcommit, system_type):
                                      )
             (clone, err) = proc.communicate()
         else:
-            print("git repository is already there, using that")
-
+            Pwarn("git repository is already there, using that",
+                  "OverwriteWarning")
```

Comment:
Nothing is overwritten, right? The code just uses the repository that already exists. I don't see another warning that is appropriate here. Maybe just leave the print statement?

Reply:
Later there will be a `checkout` and `pull` performed. Hence, if there is already one, which might be used in a different running script, it might break that, hence this warning.